testing/fzy/APKBUILD

```diff
@@ -1,14 +1,13 @@
 # Contributor: Jonathan Neuschäfer <j.neuschaefer@gmx.net>
+# Maintainer: Eivind Uggedal <eu@eju.no>
 pkgname=fzy
 pkgver=1.0
-pkgrel=0
+pkgrel=1
```

Comment:
don't think pkgrel needs to be bumped.

Reply:
Will it be built (and tested with the new acceptance tests) on the official edge builders without?

Reply:
no

Reply:
I think a revbump therefore is warranted so that it could potentially fail on the official builders in testing before being moved to community.